core/views.py

```diff
@@ -13,3 +13,12 @@ def home():
     articles = Article.objects.all()[:3]
     users = User.objects.all()[:3]
     return render_template('home.html', articles=articles, users=users)
+
+@app.errorhandler(404)
```

Comment:
- the list of possible errors: http://werkzeug.pocoo.org/docs/0.12/exceptions/

You may therefore be interested into `werkzeug.exceptions.NotFound` and `werkzeug.exceptions.InternalServerError`:

```python
from werkzeug.exceptions import NotFound, InternalServerError


@app.errorhandler(NotFound)
def error_404(e):
    return render_template('error.html', error=e, code=404), 404


@app.errorhandler(InternalServerError)
def error_500(e):
    return render_template('error.html', error=e, code=500), 500
```

However I'm not sure about the `,404` and `, 500` in the end of returns, or the the errorhandler would do it implicitely

Reply:
I'll try